dashboard/utils_test.py

```diff
@@ -451,3 +460,27 @@ def test_get_current_grade(self):
         # case when the grade is not available from edx
         with patch('edx_api.grades.models.CurrentGrades.get_current_grade', return_value=None):
             assert mmtrack.get_current_grade("course-v1:MITx+8.MechCX+2014_T1") is None
+
+    def count_courses_passed_normal(self):
```

Comment:
Can you add a test for the case where there are two passed course runs in a course?